phpBB/adm/style/acp_bbcodes.html

```diff
@@ -45,6 +45,10 @@ <h1>{L_ACP_BBCODES}</h1>
 		<dt><label for="display_on_posting">{L_DISPLAY_ON_POSTING}</label></dt>
 		<dd><input type="checkbox" class="radio" name="display_on_posting" id="display_on_posting" value="1"<!-- IF DISPLAY_ON_POSTING --> checked="checked"<!-- ENDIF --> /></dd>
 	</dl>
+	<dl>
+		<dt><label for="bbcode_icon_name">{L_BBCODE_ICON_NAME}</label><br /><span>{L_BBCODE_ICON_NAME_EXPLAIN}</span></dt>
+		<dd><input type="text" name="bbcode_icon_name" id="bbcode_icon_name" value="{{ BBCODE_ICON_NAME }}" /></dd>
+	</dl>
```

Comment:
That should not be part of the "settings" block as it's not a setting. It should have it's own legend/section, such as:
"Appearance"

Also instead of **bbcode_icon_name**, I think **bbcode_font_icon** makes more sense.